nixos/tests/all-tests.nix

```diff
@@ -1132,6 +1132,7 @@ in {
   user-home-mode = handleTest ./user-home-mode.nix {};
   ustreamer = handleTest ./ustreamer.nix {};
   uwsgi = handleTest ./uwsgi.nix {};
+  vars = handleTest ./vars.nix {};
```

Comment:
This placed seemed pretty ordered until this line got added. Or is `all-tests.nix` also just kindasorted like `module-list.nix` :)